client/multi-currency/multi-currency-settings/store-settings/index.js

```diff
@@ -0,0 +1,162 @@
+					sprintf(
+						__(
+							'Store settings allow your customers to choose which currency they ' +
+								'would like to use when shopping at your store. <learnMoreLink>' +
+								'Learn more</learnMoreLink>',
+							'woocommerce-payments'
+						),
+						LEARN_MORE_URL
+					),
+					{
+						learnMoreLink: (
+							// eslint-disable-next-line jsx-a11y/anchor-has-content
+							<a
+								href={ LEARN_MORE_URL }
+								target={ '_blank' }
+								rel={ 'noreferrer' }
+							/>
+						),
+					}
+				) }
+			</p>
+		</>
+	);
+};
+
+const StoreSettings = () => {
+	const {
+		storeSettings,
+		isLoading,
+		submitStoreSettingsUpdate,
+	} = useStoreSettings();
+	const [ isSavingSettings, setIsSavingSettings ] = useState( false );
+	const [
+		isAutomaticSwitchEnabledValue,
+		setIsAutomaticSwitchEnabledValue,
+	] = useState( false );
+
+	const [
+		isStorefrontSwitcherEnabledValue,
+		setIsStorefrontSwitcherEnabledValue,
+	] = useState( false );
+
+	useEffect( () => {
+		if ( Object.keys( storeSettings ).length ) {
+			setIsStorefrontSwitcherEnabledValue(
+				storeSettings.enable_storefront_switcher
+			);
+			setIsAutomaticSwitchEnabledValue(
+				storeSettings.enable_auto_currency
+			);
+		}
+	}, [
+		setIsAutomaticSwitchEnabledValue,
+		setIsStorefrontSwitcherEnabledValue,
+		storeSettings,
+	] );
+
+	const handleIsAutomaticSwitchEnabledClick = ( value ) => {
+		setIsAutomaticSwitchEnabledValue( value );
+	};
+
+	const handleIsStorefrontSwitcherEnabledClick = ( value ) => {
+		setIsStorefrontSwitcherEnabledValue( value );
+	};
+
+	const saveSettings = () => {
+		setIsSavingSettings( true );
+		submitStoreSettingsUpdate(
+			isAutomaticSwitchEnabledValue,
+			isStorefrontSwitcherEnabledValue
+		);
+		setIsSavingSettings( false );
+	};
+
+	return (
+		<>
+			<SettingsSection
+				Description={ StoreSettingsDescription }
+				className={ 'multi-currency-settings-store-settings-section' }
+			>
+				<LoadableBlock isLoading={ isLoading } numLines={ 10 }>
+					<Card className="multi-currency-settings__wrapper">
+						<CardBody>
+							<CheckboxControl
+								checked={ isAutomaticSwitchEnabledValue }
+								onChange={ handleIsAutomaticSwitchEnabledClick }
+								data-testid={ 'enable_auto_currency' }
+								label={ __(
+									'Automatically switch customers to their local currency if it has been enabled',
+									'woocommerce-payments'
+								) }
+							/>
+							<div className="multi-currency-settings__description">
+								{ __(
+									'Customers will be notified via store alert banner.',
+									'woocommerce-payments'
+								) }
+							</div>
+							{ 'Storefront' === storeSettings.site_theme ? (
+								<CheckboxControl
+									checked={ isStorefrontSwitcherEnabledValue }
+									onChange={
+										handleIsStorefrontSwitcherEnabledClick
+									}
+									data-testid={ 'enable_storefront_switcher' }
+									label={ __(
+										'Add a currency switcher to the cart widget',
```

Comment:
The text here appears to be the old text, it should be what's included in the Storefront integration, including the link.
https://github.com/Automattic/woocommerce-payments/blob/develop/includes/multi-currency/StorefrontIntegration.php#L58-L63

Reply:
Addressed in 69ca957.